distribution/docs/src/main/resources/_contents/_core-concepts/search-intro-contents.adoc

```diff
@@ -12,7 +12,7 @@ Additionally, any of the types can be combined to create a compound search.
 ==== Text Search
 
 A text search is used when searching for textual information.
-It searches all textual metadata fields by default, although it is possible to refine searches to a text search on a single attribute.
+It searches all textual data in the catalog, although it is possible to refine searches to a text search on a single metadata attribute.
 It is similar to a Google search over the metadata contained in the Catalog.
```

Comment:
I know this isn't part of the changeset, but is it okay to be making this Google comparison here? Could we just delete this line or should we consider making sure this usage is in line with their branding guidelines?